NEWS.md

```diff
@@ -1,3 +1,7 @@
+# covid19vaccinationch 1.1.1
+
+* added lib action r-lib/actions/setup-pandoc@v2 (#16)
```

Comment:
If feels like we want to have a more generic

```suggestion
* Maintenance of `renv` and GitHub Actions workflow (#16).
```

And add a first bullet about other things that changed in the report itself (index.Rmd diff below)

Reply:
Do you also want to

> add a first bullet about other things that changed in the report itself (index.Rmd diff below)

to the NEWS, or is it fine to just mention the technical maintenance w/o the small content change?

Reply:
No need to mention the small changed in Index.Rmd
I just took the chance to update few things to the current time, but it is not very relevant